ecl/hthor/hthor.cpp

```diff
@@ -8133,7 +8133,7 @@ CHThorDiskReadBaseActivity::CHThorDiskReadBaseActivity(IAgentContext &_agent, un
     {
         const char *recordTranslationModeHintText = _node->queryProp("hint[@name='layouttranslation']/@value");
         if (recordTranslationModeHintText)
-            recordTranslationModeHint = getTranslationMode(recordTranslationModeHintText);
+            recordTranslationModeHint = getTranslationMode(recordTranslationModeHintText, true);
```

Comment:
not directly related to this PR, but looks like hthor diskread only supports hint and not workunit option?
(and also IndexRead, Fetch and KJ)

Reply:
I will open a Jira to investigate